crates/llms/src/databricks.rs

```diff
@@ -42,6 +42,7 @@ use crate::{
 pub struct Databricks {
     pub model: String,
     client: Client<HostedModelConfig>,
+    needs_health_check: bool,
```

Comment:
nit: I would make this an enum like `HealthCheck::Required` and `HealthCheck::Skip` to make it more obvious what the booleans are when passing into the function.

Reply:
updated